packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -182,6 +182,7 @@ const Autocomplete = <
   );
 };
 
-const MemoizedAutocomplete = memo(Autocomplete) as typeof Autocomplete;
+const MemoizedAutocomplete = memo(Autocomplete);
```

Comment:
I think we still want that `typeof`. Not 100% sure, but stuff wasn't working without it before because `Autocomplete` takes generics. Maybe a React update will fix the props passthrough?